api/src/main/java/cz/muni/fi/pa165/sportsclub/facade/TeamFacade.java

```diff
@@ -0,0 +1,28 @@
+package cz.muni.fi.pa165.sportsclub.facade;
+
+import cz.muni.fi.pa165.sportsclub.dto.PlayerDto;
+import cz.muni.fi.pa165.sportsclub.dto.TeamDto;
+
+/**
+ * Facade for team
+ *
+ * @author 422636 Adam Krajcik
+ */
+public interface TeamFacade {
+
+    void createTeam(TeamDto team);
+
+    void updateTeam(TeamDto team);
+
+    void deleteTeam(TeamDto team);
+
+    void addPlayer(PlayerDto player, int jerseyNumber);
+
+    void removePlayer(PlayerDto player);
+
+    void chengeJerseyNumber(PlayerDto player);
```

Comment:
typo - change
What number to change to - I guess there should be additional parameter.